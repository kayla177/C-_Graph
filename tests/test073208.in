ENTITY IY65F9XJ name_IY65 paper
ENTITY CTOD8R6Y name_CTOD journal
ENTITY V0BCY05V name_V0BC paper
ENTITY P2GFJJXI name_P2GF journal
ENTITY P88NPQ14 name_P88N paper
ENTITY Q4Z6KRUA name_Q4Z6 journal
ENTITY SSXOWTDM name_SSXO institution
ENTITY LMKQ94PW name_LMKQ paper
ENTITY 806ZRQHM name_806Z paper
ENTITY DCCWNAAI name_DCCW conference
RELATIONSHIP LMKQ94PW published_in 806ZRQHM 4.7
RELATIONSHIP SSXOWTDM affiliated_with LMKQ94PW 7.5
RELATIONSHIP V0BCY05V authored_by P88NPQ14 10.0
RELATIONSHIP Q4Z6KRUA cited_by V0BCY05V 6.4
RELATIONSHIP IY65F9XJ authored_by LMKQ94PW 4.9
RELATIONSHIP SSXOWTDM published_in IY65F9XJ 7.9
RELATIONSHIP DCCWNAAI cited_by P2GFJJXI 2.2
RELATIONSHIP DCCWNAAI published_in V0BCY05V 6.3
RELATIONSHIP SSXOWTDM published_in P88NPQ14 7.8
RELATIONSHIP IY65F9XJ published_in P2GFJJXI 2.2
RELATIONSHIP 806ZRQHM affiliated_with P2GFJJXI 9.1
RELATIONSHIP Q4Z6KRUA collaborated_with DCCWNAAI 6.0
RELATIONSHIP CTOD8R6Y cited_by 806ZRQHM 9.1
RELATIONSHIP IY65F9XJ affiliated_with Q4Z6KRUA 2.2
RELATIONSHIP LMKQ94PW published_in V0BCY05V 9.5
PATH SSXOWTDM V0BCY05V
PRINT P88NPQ14
PRINT SSXOWTDM
DELETE CTOD8R6Y
PATH SSXOWTDM P2GFJJXI
DELETE V0BCY05V
PATH 806ZRQHM P2GFJJXI
HIGHEST
PRINT Q4Z6KRUA
PATH SSXOWTDM P2GFJJXI
EXIT
