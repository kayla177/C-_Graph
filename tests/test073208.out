success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
SSXOWTDM P88NPQ14 V0BCY05V 17.8
V0BCY05V SSXOWTDM
LMKQ94PW IY65F9XJ P88NPQ14
success
SSXOWTDM LMKQ94PW 806ZRQHM P2GFJJXI 21.3
success
806ZRQHM P2GFJJXI 9.1
P2GFJJXI P88NPQ14 29.1
DCCWNAAI IY65F9XJ
SSXOWTDM LMKQ94PW 806ZRQHM P2GFJJXI 21.3
